sdk/python/feast/registry.py

```diff
@@ -695,6 +697,80 @@ def delete_entity(self, name: str, project: str, commit: bool = True):
 
         raise EntityNotFoundException(name, project)
 
+    def apply_saved_dataset(
```

Comment:
not a blocking comment. More of an FYI

fyi @felixwang9817 since you are adding a new concept to the registry. Don't think `feast plan` could cover this since this isn't introduced at `feast apply` time. 

Also doesn't really make sense to treat this as a real FCO since the schema is a snapshot of how feature services / feature views resolve at a point in time. Doubt we'd be able to do this properly unless we have some form of stricter versioning.